src/main/java/ReviewResponse.java

```diff
@@ -2,6 +2,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.List;
+
```

Comment:
Violation of Single Responsibility Principle (SRP) as this class is now responsible for both data representation and generating a string representation. (Severity: high)